sklearn/mixture/tests/test_gmm.py

```diff
@@ -327,6 +327,26 @@ def test_1d_1component():
         assert_array_almost_equal(g.bic(X), g_full_bic)
 
 
+def test_fit_predict():
+    """
+    test that gmm.fit_predict works as expected
+    """
+    n_samples, n_dim, n_components = 100, 2, 2
+    mu = np.array([[8, 8]])
+    x0 = rng.randn(n_samples, n_dim)
+    x1 = rng.randn(n_samples, n_dim) + mu
+    x = np.vstack((x0, x1))
```

Comment:
Please use `X` instead of `x`. We have the scikit-learn specific convention to use capital variable names to represent 2D arrays in. I know this is not in line with traditional PEP8 style but we need to stay consistent with our own conventions within the project.